### DISCUSSION TITLE
November release - Annotations, Footnotes, and more

### DISCUSSION THREAD

**Discussion by taszap** [GitbookIO/community#2] [Announcements]
In November, we released annotations, footnotes, improvements to expandable blocks and some fixes. 
We created annotations because wanted you to have more ways to provide contextual information for your readers without breaking their train of thought. 

![image](https://user-images.githubusercontent.com/63643069/212049827-1089c13f-39cc-4a91-b1c6-9f09177d6c22.png)

We also added a support for footnotes in Markdown which will be imported into GitBook as inline annotations. 

**Read more** 
About our work in our [changelog](https://changelog.gitbook.com/november-2022/30-november-22-annotations-footnotes-and-more)

**What's next**
We will keep you updated on all the cool features and improvements here - until the next time!

**Comment by alialobidm** [comment]
``
